sympy/functions/special/gamma_functions.py

```diff
@@ -342,27 +342,10 @@ def _eval_rewrite_as_expint(self, s, x, **kwargs):
 
 
 class uppergamma(Function):
-    r"""
-    The upper incomplete gamma function.
-
-    It can be defined as the meromorphic continuation of
-
-    .. math::
-        \Gamma(s, x) := \int_x^\infty t^{s-1} e^{-t} \mathrm{d}t = \Gamma(s) - \gamma(s, x).
-
-    where `\gamma(s, x)` is the lower incomplete gamma function,
-    :class:`lowergamma`. This can be shown to be the same as
-
-    .. math::
-        \Gamma(s, x) = \Gamma(s) - \frac{x^s}{s} {}_1F_1\left({s \atop s+1} \middle| -x\right),
-
-    where :math:`{}_1F_1` is the (confluent) hypergeometric function.
-
-    The upper incomplete gamma function is also essentially equivalent to the
-    generalized exponential integral:
-
-    .. math::
-        \operatorname{E}_{n}(x) = \int_{1}^{\infty}{\frac{e^{-xt}}{t^n} \, dt} = x^{n-1}\Gamma(1-n,x).
+    r"""Class that represents the unevaluated upper incomplete gamma function
+    [1]_. Evaluation of the upper gamma function Γ(s, z) requires two
+    arguments, where s is the exponent of the integrand and x is the lower
+    limit of the integral. See the notes below for mathematical definitions.
```

Comment:
I'm not sure if this is how the parameters are usually named. 

In general, for mathematical functions, I don't think it's helpful to describe the parameters in plain English. It's much more useful to see the mathematical definition.